README.md

```diff
@@ -1,7 +1,7 @@
 # PyDicer: PYthon Dicom Image ConvertER
 
 [![SoftwareX](https://img.shields.io/badge/SoftwareX-10.1016/j.softx.2024.102010-green.svg)](https://doi.org/10.1016/j.softx.2024.102010)
-![Coverage](./coverage.svg)
+<!-- ![Coverage](./coverage.svg) -->
```

Comment:
```suggestion
```

I'd suggest removing this entirely only because we render the README to the docs as well and Im not sure that plugin will respect this type of comment.